docs/UserGuide.md

```diff
@@ -152,30 +152,30 @@ The **GUI** is divided into five main sections:
-| **View**     | `view INDEX`                                                                                                                                                                  |
-| **Update**   | `update c/COMPANY_INDEX app/APPLICATION_INDEX as/APPLICATION_STATUS`<br> e.g.,`update c/1 app/1 as/OA`                                                                        |
-| **Favourite**| `fav INDEX` |
-| **Unfavourite**| `unfav INDEX` |
-| **Withdraw** | `withdraw c/COMPANY_INDEX app/APPLICATION_INDEX`<br> e.g., `withdraw c/3 app/1`                                                                                               |
+| **Delete**   | `delete INDEX`                                                                                                                                                                   |
+| **List**     | `list`                                                                                                                                                                           |
+| **Edit**     | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/Meta Platforms e/jobs@meta.com`                                                        |
+| **Favourite**| `fav INDEX`                                                                                                                                                                      |
+| **Unfavourite**| `unfav INDEX`                                                                                                                                                                   |
+| **Find**     | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find Apple Inc`                                                                                                                        |
+| **View**     | `view INDEX`                                                                                                                                                                 |
+| **Apply**    | `apply INDEX n/NAME d/DESCRIPTION [as/APPLICATION_STATUS]`<br> e.g., `apply 1 n/Software Engineering Intern d/Uses React`                                                        |
+| **Update**   | `update c/COMPANY_INDEX app/APPLICATION_INDEX as/APPLICATION_STATUS`<br> e.g.,`update c/1 app/1 as/OA`                                                                       |
+| **Withdraw** | `withdraw c/COMPANY_INDEX app/APPLICATION_INDEX`<br> e.g., `withdraw c/3 app/1`                                                                                              |
+| **Reopen**   | `reopen INDEX`                                                                                                                                                                   |
```

Comment:
Think according to #298 of reorganising commands to "Company", "Application", "Misc.", Reopen belongs under Company instead.